### ISSUE TITLE
if a windows.feature does not exists then i get a rpc error

### ISSUE THREAD

**Issue by atomic111** [mondoohq/cnquery#3803]
the windows feature `AD-Domain-Servicess` does not exists

```coffee
windows.feature('AD-Domain-Servicess').installed
rpc error: code = Unknown desc = could not find feature AD-Domain-Servicess
windows.feature.installed: no data available
```

I think it would be better to show a false if a windows feature does not exists

**Comment by czunker** [comment]
I don't think we should show `false`. We don't know the state of this feature.

Let's assume someone writes the query `windows.feature('PowerShell-V2').installed`. On Windows 11 this feature is not present and we would return `false`.

But PowerShell v2 would still be active. Windows 11 exposes this information with the MQL query `windows.feature('MicrosoftWindowsPowerShellV2').installed`.

When we return `false` on the first query, you would assume PowerShell v2 is not installed.

**Comment by tas50** [comment]
Reviewed as part of a full backlog triage — **confirmed still open** in the current codebase.

providers/os/resources/windows.go:189 initWindowsServerFeature and windows.go:269/295 initWindowsOptionalFeature still `return nil, nil, errors.New("could not find feature "+name)` on an unknown feature name; comment at windows.go:279 explicitly notes it keeps the historic 'could not find feature' error behavior. No path returns installed=false for a missing feature.

**Suggested fix approach:** In providers/os/resources/windows.go, change the init functions (initWindowsServerFeature ~line 189, initWindowsOptionalFeature ~line 269/295) to return a bare resource with installed=false (set installed/name and return args,nil,nil) instead of erroring when the feature name is not found.
